mesh/ncmesh.hpp

```diff
@@ -145,7 +148,10 @@ class NCMesh
        Note that if anisotropic refinements are present in the mesh, some of the
        derefinements may have to be skipped to preserve mesh consistency. */
    virtual void Derefine(const Array<int> &derefs);
+   virtual void GetFineToCoarsePartitioning(const Array<int> &derefs,
+                                            Array<int> &new_ranks) {}
```

Comment:
This seems to be relevant only in parallel. It could be defined only in the parallel class.